src/commands/go-test.yaml

```diff
@@ -15,8 +15,6 @@ steps:
       echo -n " -X '<< parameters.pkg >>.buildTimestamp=$(date --utc '+%FT%TZ')'" >> .ldflags
   - run: |
       echo -n " -X '<< parameters.pkg >>.gitSHA=${CIRCLE_SHA1}'" >> .ldflags
-  - run: |
-      echo -n " -X '<< parameters.pkg >>.version=$(cat .project_version)'" >> .ldflags
```

Comment:
Why still have line 9 in this file? Or was it just missed?

Reply:
And 11 too, I guess

Reply:
Yeah, I missed it, the file doesn't seem to be used anywhere else so it's gone now :wink:

Reply:
Thanks for catching that!